tmt/cli.py

```diff
@@ -1032,3 +1032,23 @@ def images(context, **kwargs):
     #        we should add options to specify which provision should be
     #        cleaned, similarly to guests.
     tmt.Clean(parent=context.obj.clean, context=context).images()
+
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#  Lint
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+@main.command()
+@click.pass_context
+@click.option(
```

Comment:
Since this is exactly the same as for `lint_tests`, it would be nice to define this option only in one place and then use it as a decorator for both `lint` and `lint_tests`. There is an `tmt/options.py` file for that